pulser/parametrized/paramabc.py

```diff
@@ -12,15 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import annotations
 from abc import ABC, abstractmethod
+from typing import Dict, Any
+
+from pulser.parametrized import Variable
```

Comment:
This is what giving you the test error. Try this instead:
```suggestion
from pulser.parametrized.variable import Variable
```

Reply:
If this doesn't work either, there's another option, but try this first.

Reply:
Done

Reply:
Ok, it didn't work. Try this then:
```
from typing import Dict, Any, TYPE_CHECKING

if TYPE_CHECKING:
    from pulser.parametrized import Variable
```

Reply:
Okay, updating it.

Reply:
Done

Reply:
You have another import like this, you should change it too

Reply:
Okay